credentialdigger/client.py

```diff
@@ -559,7 +601,7 @@ def update_discovery_group(self, query, new_state, repo_url, file_name=None,
 
     def scan(self, repo_url, category=None, models=None, exclude=None,
              force=False, debug=False, generate_snippet_extractor=False,
-             local_repo=False, git_token=None):
+             similarity=False, local_repo=False, git_token=None):
```

Comment:
if you want to add a new parameter, then you also have to provide its documentation in the docstrings (of this method and the others where you pass it)